app/index.js

```diff
@@ -87,6 +87,16 @@ module.exports = class extends Generator {
 					message: 'A short description of your project:'
 				},
 				{
+					type: 'list',
+					name: 'projectType',
+					message: 'What to you want to build?',
+					choices: [
+						'A static website',
+						'A single page application'
```

Comment:
Better:
`A single page application (using plain HTML and the template engine provided by your framework)`

String must be updated in the tests as well.